nova/gce/constants.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2017 Platform9 Systems Inc.
+# All Rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either expressed or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from nova.compute import power_state
+
+GCE_STATE_MAP = {
+    "PROVISIONING": power_state.NOSTATE,
+    "STAGING": power_state.NOSTATE,
+    "RUNNING": power_state.RUNNING,
+    "STOPPING": power_state.NOSTATE,
+    "SUSPENDING": power_state.NOSTATE,
+    "SUSPENDED": power_state.SHUTDOWN,
+    "TERMINATED": power_state.CRASHED
+}
+
+GCE_FLAVOR_MAP = {
```

Comment:
Added Get machines info API which generates required Flavor map. I have fetched this through API only. Refer `gceutils.get_machines_info` for more details.